doc/man1/openssl.pod

```diff
@@ -32,6 +34,20 @@ It can be used for
  o  Handling of S/MIME signed or encrypted mail
  o  Timestamp requests, generation and verification
 
+=head2 Program Options
+
+=over 4
+
+=item B<-help> | B<--help>
+
+Provides a terse summary of all B<openssl> program options. For more detailed information, each command supports a B<-help> option.
+
+=item B<-version> | B<--version>
+
+Provides a terse summary of the B<openssl> program version. For more detailed information see L<openssl-version(1)>.
+
+=back
```

Comment:
I think this is right (e.g. use `=backl` to unindent) but I have never seen, much less auithored a .mod file.